lucet-wasi-sdk/tests/lucetc.rs

```diff
@@ -117,4 +128,39 @@ mod lucetc_tests {
         */
         let _obj = c.object_file().expect("generate code from c & d");
     }
+
+    #[test]
+    fn hello() {
+        let m = {
+            // This is like module_from_c, except missing the bits that have it leave out the wasi
+            // stdlib and tentry points:
```

Comment:
What bits? Also, `tentry` typo

Reply:
fixed this comment and the other feedback. thanks